sql/core/src/test/resources/sql-tests/inputs/literals.sql

```diff
@@ -53,6 +53,10 @@ select .e3;
 -- very large decimals (overflowing double).
 select 1E309, -1E309;
 
+-- float
+select 0F, 1F, 1.2F, -1F, -1.2F, 3.4028235E30F;
+select 3.4028235E39F;
```

Comment:
Could you move the test cases to the end of the file?

Also clearly specify the positive and negative test cases in the comments?

More test cases are needed.